README.md

```diff
@@ -16,29 +16,40 @@ Deis v2 and Helm are changing quickly. Your feedback and participation are more
 
 Hacking on Deis v2 is a bit rough right now--please help us fix the bugs and improve the installation process.
 
-First, add this Chart repo to Helm to install the "deis" chart:
+***First*** you need to have a working Kubernetes cluster on Google Container Engine (Deis, of course, works anywhere Kubernetes can be installed, but the installation instructions below are tailored to GKE, should also work on AWS)
+
+- add this Chart repo to Helm to install the "deis" chart:
+
 ```console
 $ helm repo add deis https://github.com/deis/charts
 $ helm up
 $ helm fetch deis/deis
 $ helm install deis
-$ kubectl --namespace=deis get pods  # repeat this until deis-workflow is "Running"
-$ kubectl --namespace=deis get service deis-workflow  # note the IP address for later
+$ kubectl --namespace=deis get pods -w # watch this until all pods show "Running"
+$ kubectl --namespace=deis get svc deis-router  # note the EXTERNAL IP address for later
```

Comment:
I don't think there's such a thing as an INTERNAL IP address, so maybe we should just stick with "note the IP address for later", since only the cluster IP will be shown anyways